### PR TITLE
Fix getLibraryPaths results for OSX and Linux

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ include(GitExternal)
 
 set(VERSION_MAJOR "1")
 set(VERSION_MINOR "11")
-set(VERSION_PATCH "1")
```
Comment (tribal-tec) [inline CMakeLists.txt]: 1.11.1 was not released, no bump needed imo

**Comment by tribal-tec** [comment]
I fail to see what the fix and/or the problem is

**Comment by rdumusc** [comment]
Was: Apple: LD_LIBRARY_PATH, Linux DYLD_LIBRARY_PATH
To load livre plugins, I had to set a DYLD_LIBRARY_PATH on the cluster

File: CMakeLists.txt (hunk 1 above)
Comment (rdumusc) [inline CMakeLists.txt]: No problem, as you prefer. But in general, how am I supposed to know what to do? I thought we had said that each commit which introduce a change should bump the version, either minor or patch.

**Comment by rdumusc** [comment]
updated

**Comment by tribal-tec** [comment]
Oh, blindness. Looks good! +1

**Pull request merged**